evalme/video/object_detection.py

```diff
@@ -0,0 +1,95 @@
+from copy import deepcopy
+
+from evalme.eval_item import EvalItem
+
+class VideoObjectDetectionEvalItem(EvalItem):
+    SHAPE_KEY = 'videorectanglelabels'
+
+    def extract_key_frames(self):
+        """
+        Extract frames from key frames
+        :return: List of frames
+        """
+        final_results = []
+        for result in self._raw_data:
+            sequence = result['value']['sequence']
+            if len(sequence) < 1:
+                continue
+            label = result['value'].get('labels', "")
+            sequence = sorted(sequence, key=lambda d: d['frame'])
+            if len(sequence) < 2:
```

Comment:
why do we need 2 if branches here? looks like "else" branch is enough - let's make it more concise